tests/10apidoc/03events-initial.pl

```diff
@@ -142,7 +142,7 @@ sub await_event_for
          my $found;
          foreach my $event ( @_ ) {
             not $found and $filter->( $event ) and
```

Comment:
Done 4797156. Turns out I had neatened that style of pattern up in a utility function, then managed to forget to actually use it here.